internal/kafka/command_mirror_resume.go

```diff
@@ -85,6 +85,7 @@ func printAlterMirrorResult(cmd *cobra.Command, results []kafkarestv3.AlterMirro
 				ErrorCode:             errorCode,
 				PartitionMirrorLag:    -1,
 				LastSourceFetchOffset: -1,
+				MessagesTruncated:     "0",
```

Comment:
Wait, shouldn't we just surface the `errorMessage` here instead of printing the full table? All of these `-1` values are useless as far as I can tell.

Reply:
There's not necessarily an error message if the value is `-1` - it could be that it's not relevant for the state. Also I think that issue is separate and perhaps can be dealt with in a different JIRA.